.pipelines/templates/PackageTestResultsAnalysis.yml

```diff
@@ -32,7 +32,7 @@ parameters:
     default: "$(Agent.TempDirectory)"
 
 steps:
-  - bash: sudo tdnf install -y python3-junit-xml
+  - bash: pip3 install --user junit_xml
```

Comment:
I was wondering if we should pin the version number as well. What do you think ?